src/clic.adoc

```diff
@@ -1022,12 +1019,30 @@ function pointers.
 NOTE: The hardware vectoring bit {inhv} is provided to allow resumable
 traps on fetches to the vector table.
 
-When a trap is taken, the {inhv} bit is set by hardware to indicate if {epc} is the address of a table entry
-or cleared by hardware to indicate if {epc} is the address of an instruction.

```

Comment:
get this text back into the spec, also clarify at which mode the fields are written/read